README.md

```diff
@@ -127,14 +127,15 @@ For example:
 
 ### Notes
 
-The following files are a temporary fix (leaving old paths for now):
-- logging_snippets.ipynb      
-- Neptune-API-Tour.ipynb  
-- Organize-ML-experiments.ipynb        
-- r-integration.ipynb  
-- Use-Neptune-API-to-log-your-first-experiment.ipynb
-- Monitor-ML-runs-live.ipynb  
-- neptune_test_run.ipynb  
-- pytorch_lightning-integration.ipynb  
-- Template.ipynb       
-- xgboost-integration.ipynb
\ No newline at end of file
+- The following files are a temporary fix (leaving old paths for now):
+    - logging_snippets.ipynb      
+    - Neptune-API-Tour.ipynb  
+    - Organize-ML-experiments.ipynb        
+    - r-integration.ipynb  
+    - Use-Neptune-API-to-log-your-first-experiment.ipynb
+    - Monitor-ML-runs-live.ipynb  
+    - neptune_test_run.ipynb  
+    - pytorch_lightning-integration.ipynb  
+    - Template.ipynb       
+    - xgboost-integration.ipynb
+- XGBoost integration (upgraded libs case) is not tested on Win with Python 3.8, as it causes [tkinter error](https://github.com/neptune-ai/neptune-examples/runs/1309037471?check_suite_focus=true).
```

Comment:
@kamil-kaczmarek we should mention why the "how-it-works product tour" was excluded in a similar fashion.

Reply:
done.